yt/frontends/amrvac/tests/test_outputs.py

```diff
@@ -11,7 +11,7 @@
 from yt.units import YTQuantity
 
 blastwave_spherical_2D = "amrvac/bw_2d0000.dat"
-khi_cartesian_2D = "amrvac/kh_2D0000.dat"
+khi_cartesian_2D = "amrvac/kh_2d0000.dat"
```

Comment:
the fact that 2d and 3D use different conventions is confusing.

Reply:
indeed. The AMRVAC sample files are straight from AMRVAC's testsuite which _doesn't_ have naming conventions other than the suffix's shape `[0-9]{4}.dat`